dump-objects

```diff
@@ -240,7 +244,7 @@ fetch_objects() {
     local errprefix=
 
     if verbose; then
```

Comment:
To get this output I'd have to set verbose and debug now? Wouldn't it make sense to deprecate the verbose flag?